blocks/init/src/Blocks/components/layout-three-columns/layout-three-columns.php

```diff
@@ -51,7 +52,12 @@
 
 ?>
 
-<<?php echo esc_attr($layoutThreeColumnsHtmlTag); ?> class="<?php echo \esc_attr($layoutClass); ?>">
+<<?php echo esc_attr($layoutThreeColumnsHtmlTag); ?>
+	class="<?php echo \esc_attr($layoutClass); ?>"
+	<?php if ($layoutThreeColumnsAriaRole) { ?>
+		role="<?php echo \esc_attr($layoutThreeColumnsAriaRole); ?>"
```

Comment:
Is `role` attribute something that can be left with an empty or some default value?
I'm thinking that this conditional can be written with a ternary or null coalesce operators.

It will look a bit cleaner IMO.

Reply:
It should fall back to the implicit role if empty. However, the ARIA specification advises against adding redundant `role` attributes, which is why I used this. 

> The explicit semantic role of an element is determined by its role attribute (if any).
> 
> The role attribute takes a list of tokens. The explicit semantic role is the first valid role in this list. The valid roles are all non-abstract roles from WAI-ARIA Specifications. If the element has no role attribute, or if it has one with no valid role, then this element has no explicit semantic role.
> 
> The implicit semantic role of an element is a pre-defined value given by the host language which depends on the element and its ancestors.
> 
> (from https://www.w3.org/WAI/standards-guidelines/act/rules/role-attribute-valid-value-674b10/)